README.md

```diff
@@ -23,7 +23,7 @@ https://developer.jboss.org/wiki/PerformanceGatesForAcceptingPerformanceFixesInN
 
 Requirements
 ------------
-To build this project you will need a JDK (Java Development Kit) with a minimum version of 11.
+To build this project you will need a JDK (Java Development Kit) with a minimum version of 11 and a Maven with a minimum version specified in `.mvn/wrapper/maven-wrapper.properties`
```

Comment:
Instead of using the markdown syntax for code why not directly link to the file: `... specified in the [maven wrapper properties file](.mvn/wrapper/maven-wrapper.properties)`.

Reply:
> Instead of using the markdown syntax for code why not directly link to the file: `... specified in the [maven wrapper properties file](.mvn/wrapper/maven-wrapper.properties)`.

changes updated.